packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -343,12 +323,7 @@ class DropdownCore extends React.Component<Props, State> {
         if (open) {
             // Reset focused index
             this.focusedIndex = initialFocusedIndex;
-            // We explicitly set focus to the first item only if we sense
-            // that the user opened the menu via the keyboard
-            if (keyboard) {
-                this.keyboardNavOn = true;
-                this.scheduleToFocusCurrentItem();
-            }
+            this.scheduleToFocusCurrentItem();
```

Comment:
It seems that this is called again in line # 333. Could you please get rid of one of these fn calls?